### ISSUE TITLE
Form or join a union!

### ISSUE THREAD

**Issue by twome** [shengxinjing/programmer-job-blacklist#289]
Workers create everything that society needs, not bosses. The owners of these gigantic technology companies are mostly absent from the entire process of making apps, or designing chipsets. That includes the board of directors as well as all the shareholders. They profit by paying each worker less than that worker earns for the company – they are parasites and autocrats.

Because workers do all the work, that means they have the power to *stop* all the work. Bargaining with the capitalist class as one big group of people gives you **so** much more power than as an individual. The threat of striking is the #1 thing that the property-owner class fears. It's by far the best way to guarantee more rights for working-class programmers. Please look for a suitable union that you can join, or create a new one together! Solidarity from Australia (the government here has basically outlawed worker's strikes! 👎)

❤️ ✊

**Comment by ayla7723** [comment]
I agree with you 100%. It's crazy how many people don't really care about unionizing meanwhile the gap between rich and poor is worse today than it was in the Guilded Age.
